Sources/FoundationEssentials/Predicate/Expressions/Range.swift

```diff
@@ -65,3 +65,106 @@ extension PredicateExpressions.Range : Codable where LHS : Codable, RHS : Codabl
 
 @available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
 extension PredicateExpressions.Range : Sendable where LHS : Sendable, RHS : Sendable {}
+
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
+extension PredicateExpressions {
+    public struct ClosedRange<
```

Comment:
I decided to delete the separate `ClosedRange.swift` file and just consolidate all range operators into this file